northstar/src/runtime/config.rs

```diff
@@ -35,7 +28,7 @@ pub struct Config {
     pub cgroup: NonNulString,
     /// Console configuration
     #[serde(deserialize_with = "console")]
-    pub consoles: HashMap<Url, Console>,
+    pub consoles: HashMap<Url, ConsoleQoS>,
```

Comment:
Configuring the rate limit on external connections is optional - but very low hanging :-)